tools/tests/old/test-watch.js

```diff
@@ -1,3 +1,5 @@
+require('babel/register')(require('../../babel-config.js'));
```

Comment:
Yup. I decided not to worry about this too much because these are the "old" test files which will hopefully one day not exist, and be folded into the new tests.